### ISSUE TITLE
[Enhancement] Rockstar Games Launcher accomplishments

### ISSUE THREAD

**Issue by FivesBlue** [Lacro59/playnite-successstory-plugin#373]
Hi, I've noticed that SuccessStory either supports or plans to support achievements from nearly every game DRM that has them, except for Rockstar Games Launcher. This being the case, I though I'd make this suggestion. Having Rockstar accomplishments as a feature in SuccessStory I think would be a fantastic addition to the plugin!
Thanks for reading :)

**Comment by PatrikPepega** [comment]
Agree. It would be a great thing. Especially for people that owns Gta 5 on Rockstar Games.
